common/api/core-geometry.api.md

```diff
@@ -5502,6 +5502,47 @@ export class Segment1d {
     x1: number;
 }
 
```

Comment:
I dont see these function to be newly added so they were internal and you just made them public. ok good!